### PR TITLE
Update RSC page

### PR DESCRIPTION
**Pull request by Hwurzburg**
So rather than pushing into your repo, Bill, since I squashed all the intervening fixs I didnt want to mess with your repo history...
so you can see the result before its merged into master...

-I squashed into one commit your preceeding stuff
-fixed formatting and broke up some paragraphs...I've built the wiki and it looks good now...
-added a little to the explanations so its clear when we are talking about throttle outputs,etc.

I think this will be a little clearer to heli newbies as well as us old timers that have been out of the loop a while...hopefully...
we can just merge this if you are okay....keeps your commit credit intact also...

### CODE REVIEW

**Comment by bnsgeyer** [comment]
Other than the comment above, I think it looks good

File: copter/source/docs/traditional-helicopter-rsc-setup.rst [hunk 1]
```diff
@@ -4,16 +4,40 @@
 Rotor Speed Control Setup
 =========================
 
-Rotor speed controller set up is described in video 3
-for electric powered helicopters and video 4 for piston 
-powered helicopters on the following page.
+The Rotor Speed Control (RSC) uses the throttle output (``SERVOx_FUNCTION`` = 70) to the engine to control rotor speed. This can be done in a few different ways:
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply copter/source/docs/traditional-helicopter-rsc-setup.rst]: aha! light bulb shines.....need to look thru everything and make sure HeliRSC is used as output name!

[the remark this answers is not in the document]
Comment (Hwurzburg) [reply copter/source/docs/traditional-helicopter-rsc-setup.rst]: just pushed a fix...and a few "throttles" got changed to heliRSC ....merging

**Comment by Hwurzburg** [comment]
merged...thanks!

**Pull request merged**